include/umf/memory_provider_ops.h

```diff
@@ -91,8 +91,9 @@ typedef struct umf_memory_provider_ops_t {
     ///        result in string representation
     /// @param pError [out] pointer to an integer where the adapter specific error code will be stored
     ///
-    void (*get_last_native_error)(void *provider, const char **ppMessage,
-                                  int32_t *pError);
+    umf_result_t (*get_last_native_error)(void *provider,
```

Comment:
missing @return here and in other places

Reply:
done